test/Spring-test.js

```diff
@@ -351,6 +351,9 @@ describe('Spring', () => {
       },
 
       componentDidMount() {
+        this.update();
```

Comment:
the rule is `Do not use setState in componentDidMount  react/no-did-mount-set-state` are you sure?